frontend/public/components/_service-class.scss

```diff
@@ -0,0 +1,12 @@
+.co-service-class-title {
+  text-align: center;
```

Comment:
I think left-aligned would be better. Does it make sense just to use an `<h2>` for the title?